File: SWEA/s1206.java
import java.util.*;

public class s1206 {
    public static void main(String[] args) {
        int[] arr = new int[1000];
        Scanner sc = new Scanner(System.in);
        for(int i=1; i<=10; i++) {
            int tc = sc.nextInt();

            int ans = 0;
            for(int j=0; j<tc; j++) {
                int data = sc.nextInt();
                arr[j] = data;

            }

            for(int j=2; j<tc-2; j++) {
                if(arr[j] > Math.max(arr[j+1], arr[j+2]) && arr[j] > Math.max(arr[j-1], arr[j-2])) {
                    ans += arr[j] - Math.max(Math.max(arr[j + 1], arr[j + 2]), Math.max(arr[j - 1], arr[j - 2]));
                }
            }

            System.out.println("#"+i+" "+ans);
        }
    }

}

File: SWEA/s1206.cpp
// https://swexpertacademy.com/main/code/problem/problemDetail.do?contestProbId=AV134DPqAA8CFAYh

#include <iostream>
#include <algorithm>
using namespace std;

int arr[1000];

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int tc = 0;
    for(int i=1; i<=10; i++) {
        cin >> tc;

        int ans = 0;
        for(int j=0; j<tc; j++) {
            cin >> arr[j];
        }

        for(int j=2; j<tc-2; j++) {
            if(arr[j] > max(arr[j+1], arr[j+2]) && arr[j] > max(arr[j-1], arr[j-2])) {
                ans += arr[j] - max(max(arr[j + 1], arr[j + 2]), max(arr[j - 1], arr[j - 2]));
            }
        }

        cout << "#" << i << " " << ans << '\n';
    }

    return 0;
}